mtgjson5/price_builder.py

```diff
@@ -74,9 +75,12 @@ def build_today_prices() -> Dict[str, Any]:
     tcgplayer = _generate_prices(TCGPlayerProvider())
     card_market = _generate_prices(CardMarketProvider())
     card_kingdom = _generate_prices(CardKingdomProvider())
+    cardsphere = _generate_prices(MultiverseBridgeProvider())
```

Comment:
Naming is not consistent. Should either rename provider class, or rename variable to match

Reply:
After further review, would side with renaming the variable over renaming the class

Reply:
The naming is intentionally different, as MB can/will provide other price Ids in the future, and this sets a precedent that we can pull it in multiple ways. If my logic here is wrong, I'm fine with changing it

Reply:
Oh, makes enough sense. Then yeah, keeping it is fine 👍